tests/datasource/fluent/test_spark_azure_blob_storage_datasource.py

```diff
@@ -414,10 +417,12 @@ def instantiate_azure_client_spy(self) -> None:
         instantiate_azure_client_spy,
         raising=True,
     )
+    asset_specified_metadata = {"asset_level_metadata": "my_metadata"}
```

Comment:
This test and our other cloud storage integration tests are xfailed currently but I added these none-the-less. If there is something I should do to manually verify this (or someone I should ask who has these backends setup) let me know.

We should probably store credentials in our secret store and not xfail these.

Reply:
@billdirks This was the right thing to do -- thank you -- they are X-failed because the data is unreachable with the current state of the remote store mocks, but we will figure it out eventually, so good to keep these tests ready to go when we can remove the X-Fail directives.  Thanks.